CRM/Upgrade/Incremental/sql/4.7.alpha1.mysql.tpl

```diff
@@ -19,3 +19,17 @@ UPDATE civicrm_option_value SET {localize field="label"}label = 'Activity Detail
 
 UPDATE civicrm_option_value SET {localize field="label"}label = 'Activity Summary Report'{/localize}
   WHERE value = 'activitySummary' AND option_group_id = @option_group_id_report;
+
+-- Added is_payment flag 
+ALTER TABLE civicrm_financial_trxn ADD COLUMN is_payment tinyint(4) DEFAULT '0' COMMENT 'Is this entry either a payment or a reversal of a payment?';
+
+-- Add payments user dashboard option
+
+SELECT @option_group_id_dash := MAX(id) from civicrm_option_group where name = 'user_dashboard_options';
+SELECT @option_group_id_dash_wt := MAX(weight) FROM civicrm_option_value WHERE option_group_id = @option_group_id_dash;
+SELECT @option_group_id_dash_val := MAX(CAST( `value` AS UNSIGNED )) FROM civicrm_option_value WHERE option_group_id = @option_group_id_dash;
+
+INSERT INTO
+`civicrm_option_value` (`option_group_id`, {localize field='label'}`label`{/localize}, `value`, `name`, `grouping`, `filter`, `is_default`, `weight`, {localize field='description'}`description`{/localize}, `is_optgroup`, `is_reserved`, `is_active`, `component_id`, `visibility_id`)
+VALUES
+(@option_group_id_dash, {localize}'{ts escape="sql"}Payments{/ts}'{/localize}, @option_group_id_dash_val+1, 'Payments', NULL, NULL, NULL, @option_group_id_dash_wt+1, NULL, 0, 0, 1, NULL, NULL);
```

Comment:
Hmm, I checked the other instances of adding option values to sql install files but it doesn't look like the 'value' field is localized. It's just the label and description fields that are localized.